indy_client/test/cli/test_pool_upgrade_schedule.py

```diff
@@ -19,7 +19,7 @@ def validUpgrade(_validUpgrade):
     return _validUpgrade
 
 
-def testPoolUpgradeScheduledOnProperDate(poolNodesStarted,
+def testNodeUpgradeScheduledOnProperDate(poolNodesStarted,
```

Comment:
Why did we rename it? We test POOL_UPGRADE here, don't we?

Reply:
The module name is `test_pool_upgrade_schedule` since we test the pool upgrade schedule here.
For each node upgrade is scheduled on individual date/time. The test verifies that for each node the upgrade is scheduled on the proper date, so its name is `testNodeUpgradeScheduledOnProperDate`.